lang/cs/Org.Apache.REEF.IO.TestClient/HadoopFileInputPartitionTest.cs

```diff
@@ -55,6 +55,7 @@ internal static bool TestWithByteDeserializer()
                     .Set(FileSystemInputPartitionConfiguration<IEnumerable<byte>>.FilePathForPartitions, remoteFilePath1)
                     .Set(FileSystemInputPartitionConfiguration<IEnumerable<byte>>.FilePathForPartitions, remoteFilePath2)
                     .Set(FileSystemInputPartitionConfiguration<IEnumerable<byte>>.FileSerializerConfig, serializerConfString)
+                    .Set(FileSystemInputPartitionConfiguration<IEnumerable<byte>>.TempFileFolder, @".\reef\mytest\")
```

Comment:
@markusweimer , I remember in our discussions you saying to implement a provider for this then add TempFolder the way it is being done above. Any reason that approach is preferable over what's done above by Julia?